src/server.h

```diff
@@ -261,9 +261,6 @@ extern int configOOMScoreAdjValuesDefaults[CONFIG_OOM_COUNT];
 #define CLIENT_LUA_DEBUG_SYNC (1<<26)  /* EVAL debugging without fork() */
 #define CLIENT_MODULE (1<<27) /* Non connected client used by some module. */
 #define CLIENT_PROTECTED (1<<28) /* Client should not be freed for now. */
-#define CLIENT_PENDING_READ (1<<29) /* The client has pending reads and was put
-                                       in the list of clients we can read
-                                       from. */
```

Comment:
This will cause "blame" mess. It think it's better not to collapse but rather re-use the space next time we need a flag. Note that using an enum would have solved this issue.

Reply:
ok, so let's put some placeholder line here to make it visible that 29 is free.
this line should somehow stand out (maybe by being shorter, or have no comment)

Reply:
done.